CMakeLists.txt

```diff
@@ -11,10 +11,20 @@ if (NOT CMAKE_BUILD_TYPE)
 endif()
 
 
+# FIXME This was set in a XL-related commit:
+# https://github.com/kokkos/kokkos-fortran-interop/commit/f232302284400a90410a36edb14f538c20b8a71b
+# We need to check if it is still necessary
 set(BUILD_SHARED_LIBS OFF)
 option(FLCL_BUILD_EXAMPLES "Build examples" ON)
 option(FLCL_BUILD_TESTS "Build tests" ON)
 
+# The tests and the examples need to be built with PIE because they link against GLICXX
+if (${FLCL_BUILD_TESTS} OR ${FLCL_BUILD_TESTS})
+  if (NOT DEFINED CMAKE_POSITION_INDEPENDENT_CODE OR NOT ${CMAKE_POSITION_INDEPENDENT_CODE})
+    message(FATAL_ERROR "Enabling the tests or the examples require configuring with -DCMAKE_POSITION_INDEPENDENT_CODE=ON")
```

Comment:
Is that not something we could set as a property on our executables?

Reply:
Yes, this is just temporary. I added the error message because I needed that flag to compile the tests. Once I've set up the CI, I will update the build system.

Reply:
Fine